frontend/catalyst/jax_primitives.py

```diff
@@ -76,7 +76,7 @@
 from catalyst.utils.extra_bindings import FromElementsOp, TensorExtractOp
 from catalyst.utils.types import convert_shaped_arrays_to_tensors
 
-# pylint: disable=unused-argument,abstract-method,too-many-lines
+# pylint: disable=unused-argument,too-many-lines
```

Comment:
same here, I'm getting tons of them which is why i added it

Reply:
```suggestion
# pylint: disable=unused-argument,abstract-method,too-many-lines
```